### PR TITLE
feat(whats-new): add fedramp logs WN post (publish 07-14-22)

### PR DESCRIPTION
**Pull request by ymirelesNewRelic**
FedRAMP compliant accounts will start collecting application logs in context automatically, starting August 12th

### CODE REVIEW

**Review by aalfano7 — COMMENTED** [review]
I would rephrase the first sentence into two sentences:

On August 12th, 2022, accounts approved under the Federal Risk and Authorization Management Program (FedRAMP) currently using one of our [supported APM Agents](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/get-started-logs-context#agents) will automatically start collecting and forwarding logs to New Relic. This provides you with the logs you need to troubleshoot issues faster, making your telemetry data more valuable without the need for forwarders or custom solutions.


I would also update the last sentences to read like this:

Collecting application logs means that more data will be ingested into the platform, at your standard ingest rate. **You always have direct control over which data you send to New Relic by [configuring your agent](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/get-started-logs-context/#agents).** 

See our documentation about [disabling automatic logging](https://docs.newrelic.com/docs/logs/logs-context/disable-automatic-logging/) or [New Relic's data management hub](https://docs.newrelic.com/docs/logs/logs-context/disable-automatic-logging/#disable-ui) for more information on how to adjust your default settings.

**Comment by rhetoric101** [comment]
@aalfano7 I took your editing suggestions and created a new revision. You can see the preview [here](https://docswebsitedevelop-ymirelesnewrelicdocswebsi15193.gtsb.io/whats-new/2022/07/whats-new-07-14-fedramp-logs-update). I noticed that the link to the data management hub looked incorrect, so I changed it to this: https://docs.newrelic.com/docs/data-apis/manage-data/manage-your-data/
